apps/src/blockly/customBlocks/googleBlockly/mixins/procedureCallerOnChangeMixin.ts

```diff
@@ -20,7 +24,7 @@ const procedureCallerOnChangeMixin = {
       Blockly.isEmbeddedWorkspace(this.workspace)
     )
       return;
-    if (event.type === Blockly.Events.BLOCK_MOVE) this.updateArgsMap_(true);
+    if (event.type === Blockly.Events.BLOCK_MOVE) this.updateArgsMap_();
```

Comment:
`updateArgsMap_` [does not take any parameters](https://github.com/google/blockly-samples/blob/498c36e7f7596b1577a46c58db3ca1ed0002cc6a/plugins/block-shareable-procedures/src/blocks.ts#L1133)